components/script/dom/webidls/Bluetooth.webidl

```diff
@@ -23,9 +23,8 @@ interface Bluetooth {
 //  [SecureContext]
 //  readonly attribute BluetoothDevice? referringDevice;
 //  [SecureContext]
-//  Promise<BluetoothDevice> requestDevice(RequestDeviceOptions options);
   [Throws]
```

Comment:
Specifically, the [Throws] annotation, since none of the methods throw an exception any more.